FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -204,12 +204,18 @@ PARAMETERs are: [ ] = optional; < > = user supplied value
             (ii) you want to be able to run some flag in recon-all, without also regenerating the surfaces.
                  e.g., [--existing-subject --extra-reconall-arg=-show-edits --no-conf2hires]
 
+  [--mpp-mode={HCPStyleData, LegacyStyleData}]
+      Indicates, which variant of MPP to use. "HCPStyleData" (the default) follows the processing steps 
+         described in Glasser et al. (2013) and requires 'HCP-Style' data acquistion. "LegacyStyleData" 
+         allows additional processing functionality and use of some acquisitions that do not conform 
+         to 'HCP-Style' expectations in this case missing high-resolution T2w image 
```

Comment:
Awkward phrasing. Better would be: `to 'HCP-Style' expectations; i.e., in this case, missing high-resolution T2w image`

Reply:
ok